src/Services/AddThisScriptManager.php

```diff
@@ -191,7 +193,7 @@ private function getJsAddThisConfig() {
       'ui_open_windows' => $config->get('compact_menu.menu_style.addthis_open_windows_enabled'),
       'ui_use_css' => $config->get('compact_menu.additionals.addthis_standard_css_enabled'),
       'ui_use_addressbook' => $config->get('compact_menu.additionals.addthis_addressbook_enabled'),
-      'ui_language' => $language->language,
+      'ui_language' => isset($language) ? $language->language : "und",
```

Comment:
In D8, the language none has changed to `LANGCODE_NOT_SPECIFIED`. See, https://api.drupal.org/api/drupal/core%21lib%21Drupal%21Core%21Language%21LanguageInterface.php/constant/LanguageInterface%3A%3ALANGCODE_NOT_SPECIFIED/8